include/event_object_movement.h

```diff
@@ -16,14 +16,22 @@ enum SpinnerRunnerFollowPatterns
     RUNFOLLOW_SOUTH_EAST_WEST
 };
 
+enum ReflectionTypes
+{
+    REFL_TYPE_NONE,
+    REFL_TYPE_ICE,
+    REFL_TYPE_WATER
+};
+#define NUM_REFLECTION_TYPES 2
```

Comment:
This should be part of the enum, `reflectionFlags` can do `NUM_REFLECTION_TYPES - 1` (it's just excluding `REFL_TYPE_NONE`)